source/Grid/Grid.js

```diff
@@ -979,16 +979,20 @@ export default class Grid extends PureComponent {
    * This prevents jerky/stuttery mouse-wheel scrolling.
    */
   _debounceScrollEnded () {
-    const { scrollingResetTimeInterval } = this.props
-
     if (this._disablePointerEventsTimeoutId) {
-      clearTimeout(this._disablePointerEventsTimeoutId)
+      window.cancelAnimationFrame(this._disablePointerEventsTimeoutId)
     }
 
-    this._disablePointerEventsTimeoutId = setTimeout(
-      this._debounceScrollEndedCallback,
-      scrollingResetTimeInterval
-    )
+    const delay = () => {
+      if (Date.now() - this._scrollDebounceStart >= this.props.scrollingResetTimeInterval) {
```

Comment:
We could assign `scrollingResetTimeInterval` to a local variable and avoid the `this.props.` lookup chain.